Makefile

```diff
@@ -189,6 +190,18 @@ ifdef DARWIN
 		$@
 endif
 
+$(OUTPUT_DIR)/libs/libkoreader-nnsvg.so: nnsvg.c \
+			$(if $(USE_LUAJIT_LIB),$(LUAJIT_LIB),) \
+			$(NANOSVG_HEADERS)
+	$(CC) -I$(NANOSVG_INCLUDE_DIR) \
+	$(CFLAGS) $(DYNLIB_CFLAGS) -fvisibility=hidden -Wall -o $@ nnsvg.c
```

Comment:
@poire-z Wrt https://github.com/koreader/koreader-base/pull/1250#issuecomment-739471075 what if you stick a `-lm` in there?

Reply:
I don't have (not want :/) any android build tools. So, if you do, and you get the same crash as on phones, what if ? :)

Reply:
I spent the past 15-20 minutes building for Android as I haven't since… oh, October or so when I didn't have any free time at all due to moving, so now I'll be able to do some quick tests. 

I only have to comment out these lines to test?
```lua
-- Uncomment to use SVG icons from one of these directories
-- ICONS_ALT_SVG_DIR = "resources/icons/src/"
-- ICONS_ALT_SVG_DIR = "resources/icons/svg/"
```

Reply:
Yes, the /src/ one where we already have a few SVGs (it will look ugly, some will be unparsable, black...).

For testing https://github.com/koreader/koreader/pull/6937, you can just download the zip from https://github.com/johnbeard/koreader/tree/icons , and extract the svg from its resources/icons/src/ into that directory (I put them into "resources/icons/svg/", so it's clean to update).

Reply:
Yup, -lm gets around that error. But it looks like we do indeed need to link luajit too:

```
12-06 11:41:46.724  4000  4030 E KOReader: 	dlopen failed: cannot locate symbol "luaL_checklstring" referenced by "/data/data/org.koreader.launcher/files/libs/libkoreader-nnsvg.so"...
```

Reply:
I.e., works like this, but the -lluajit should be behind a `$(if $(USE_LUAJIT_LIB),$(LUAJIT_LIB),)` type check.
```diff
diff --git a/Makefile b/Makefile
index b49b784e..751c8f71 100644
--- a/Makefile
+++ b/Makefile
@@ -194,7 +194,7 @@ $(OUTPUT_DIR)/libs/libkoreader-nnsvg.so: nnsvg.c \
                        $(if $(USE_LUAJIT_LIB),$(LUAJIT_LIB),) \
                        $(NANOSVG_HEADERS)
        $(CC) -I$(NANOSVG_INCLUDE_DIR) \
-       $(CFLAGS) $(DYNLIB_CFLAGS) -fvisibility=hidden -Wall -o $@ nnsvg.c
+       $(CFLAGS) $(DYNLIB_CFLAGS) -lm -lluajit -fvisibility=hidden -Wall -o $@ nnsvg.c
 ifdef DARWIN
        install_name_tool -change \
                `otool -L "$@" | grep "libluajit" | awk '{print $$1}'` \
```

Reply:
![Screenshot_2020-12-06_11-48-17](https://user-images.githubusercontent.com/202757/101278041-f17ddd80-37b8-11eb-8dea-8ce67b724e3e.png)